doc/classes/LinkButton.xml

```diff
@@ -51,13 +56,22 @@
 			Set additional options for BiDi override.
 		</member>
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
-			The button's text that will be displayed inside the button's area.
+			Sets the label to the specified text.
```

Comment:
It's still a button, even if it has the appearance of a link :slightly_smiling_face: 

```suggestion
			The button's text that will be displayed inside the button's area.
```

Reply:
Sorry i wasn't supposed to override these, just messed up when merging conflicts 😢 (Not giving up!)